svn/common_base.py

```diff
@@ -17,12 +18,24 @@ def external_command(self, cmd, success_code=0, do_combine=False,
         env['LANG'] = svn.config.CONSOLE_ENCODING
         env.update(environment)
 
-        p = subprocess.Popen(
-            cmd,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
-            cwd=wd,
-            env=env)
+        if sys.platform.lower().startswith('win'):
```

Comment:
`subprocess.STARTF_USESHOWWINDOW` is only defined in Windows. Just try to get it and handle `AttributeError` if it doesn't exist. We don't want to get involved with detecting platform names.

Reply:
Done.